src/operators.jl

```diff
@@ -398,15 +418,45 @@ function _multiply!{T<:Union{GenericAffExpr,GenericQuadExpr}}(ret::Array{T}, lhs
     ret
 end
 
+# for transposed multiplication
+function _multiplyt!{T<:Union{GenericAffExpr,GenericQuadExpr}}(ret::Array{T}, lhs::Matrix, rhs::SparseMatrixCSC)
+    rowval = rhs.rowval
+    nzval  = rhs.nzval
+    for multivec_row ∈ 1:size(lhs,2) # transpose
+        for col ∈ 1:rhs.n
+            idxset = rhs.colptr[col]:(rhs.colptr[col+1]-1)
+            q = ret[multivec_row, col]
+            _sizehint_expr!(q, length(idxset))
+            for k ∈ idxset
+                append!(q, lhs[rowval[k], multivec_row] * nzval[k]) # transpose
+            end
+        end
+    end
+    ret
+end
+
+
 # TODO: implement sparse * sparse code as in base/sparse/linalg.jl (spmatmul)
 _multiply!{T<:JuMPTypes}(ret::AbstractArray{T}, lhs::SparseMatrixCSC, rhs::SparseMatrixCSC) = _multiply!(ret, lhs, full(rhs))
+_multiplyt!{T<:JuMPTypes}(ret::AbstractArray{T}, lhs::SparseMatrixCSC, rhs::SparseMatrixCSC) = _multiplyt!(ret, lhs, full(rhs))
 
 _multiply!(ret, lhs, rhs) = A_mul_B!(ret, lhs, ret)
 
 (*){T<:JuMPTypes}(             A::Union{Matrix{T},SparseMatrixCSC{T}}, x::Union{Matrix,   Vector,   SparseMatrixCSC})    = _matmul(A, x)
 (*){T<:JuMPTypes,R<:JuMPTypes}(A::Union{Matrix{T},SparseMatrixCSC{T}}, x::Union{Matrix{R},Vector{R},SparseMatrixCSC{R}}) = _matmul(A, x)
 (*){T<:JuMPTypes}(             A::Union{Matrix,   SparseMatrixCSC},    x::Union{Matrix{T},Vector{T},SparseMatrixCSC{T}}) = _matmul(A, x)
 
+import Base.At_mul_B
+import Base.Ac_mul_B
+# these methods are called when one does A.'*v or A'*v respectively 
+At_mul_B{T<:JuMPTypes}(A::Union{Matrix{T},SparseMatrixCSC{T}}, x::Union{Matrix, Vector, SparseMatrixCSC}) = _matmult(A, x)
+At_mul_B{T<:JuMPTypes,R<:JuMPTypes}(A::Union{Matrix{T},SparseMatrixCSC{T}}, x::Union{Matrix{R}, Vector{R}, SparseMatrixCSC{R}}) = _matmult(A, x)
+At_mul_B{T<:JuMPTypes}(A::Union{Matrix,SparseMatrixCSC}, x::Union{Matrix{T}, Vector{T}, SparseMatrixCSC{T}}) = _matmult(A, X)
```

Comment:
The argument to the function is ``x`` (lowercase), while the argument you pass to ``_matmult`` is ``X`` (uppercase).